main.nf

```diff
@@ -166,6 +171,11 @@ if( workflow.profile == 'uppmax' || workflow.profile == 'uppmax-modules' || work
     if ( !params.project ) exit 1, "No UPPMAX project ID found! Use --project"
 }
 
+//AWSBatch sanity checking
+if(workflow.profile == 'awsbatch'){
+    if (!params.awsqueue || !params.awsregion || !params.workDir.startsWith('s3') || !params.outdir-startsWith('s3')) exit 1, "Specify AWS Batch required parameters!"
```

Comment:
Maybe split this up a bit with different error messages - I could imagine getting super frustrated if I did specify all parameters but used a `http` address for the outdir or something ;)

Also, minor typo: hyphen instead of a dot: `!params.outdir-startsWith('s3')`

Reply:
Found the typo - already fixed, but I can break it up yes!